wordpress-mobile/WordPress-iOS
# Analytics: Migrates session count tracking to WPAppAnalytics.

Closes #6398
Closes #3515

This PR migrates session count handling from the mixpanel tracker into WPAppAnalytics, preserving the existing session count.

To test:
Scenario 1 - Upgrade
- Begin on `develop` launch the app and background it before stopping the debugger so mixpanel super properties are preserved.  The session count stored in super properties should be greater than zero.
- Switch to this branch.  Add break points to observe the migration of the session count, and where the application installed event is dispatched.
- Run the branch.  Confirm that the value is properly migrated.  Confirm that the application installed event is *not* dispatched.

Scenario 2 - Install / First run
- Delete the app from the simulator.  No really, actually delete the app.
- Run the app. 
- Confirm application installed event is dispatched.
- Confirm the session count is 1.

Scenario 3 - Subsequent run.
- Relaunch the app.
- Confirm the application installed event is *not* dispatched.
- Confirm the session count was incremented.

Needs review: @astralbodies
